CognitiveConcierge-iOS/CognitiveConcierge/EntertainmentOptionsViewController.swift

```diff
@@ -62,8 +62,12 @@ class EntertainmentOptionsViewController: UIViewController {
         convoContext = nil
         
         // Instantiating to start first conversation with Watson.
-        convoService = Conversation(username: configuration?["ConversationUsername"] as! String, password: configuration?["ConversationPassword"] as! String, version: configuration?["ConversationVersion"] as! String)
-        
+        if let iamApiKey = configuration?["ConversationApiKey"] as? String, !iamApiKey.isEmpty {
```

Comment:
LGTM

Reply:
Small problem: here I refer to "ConversationApiKey" but in the plist it's "ConversationIamApiKey". I'll change it to "ConversationApiKey" in the plist.